www-apps/prewikka/prewikka-4.0.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+# Prewikka support python 3 but not twisted-names so we
+# can't provide python 3 for now for prewikka
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Graphical front-end analysis console for the Prelude Framework"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="adns"
+
+PYTHON_REQ_USE="ssl"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/lesscpy[${PYTHON_USEDEP}]
+	sys-devel/gettext"
+
+RDEPEND="dev-python/Babel[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pytz[${PYTHON_USEDEP}]
+	dev-python/mako[${PYTHON_USEDEP}]
+	dev-python/werkzeug[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	adns? ( dev-python/twisted-names[${PYTHON_USEDEP}] )
```

Comment:
Split twisted packages are the old versions. Please try if it works with modern `dev-python/twisted`.